jdtls.ext/com.shengchen.checkstyle.checker/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>com.shengchen.checkstyle</groupId>
+		<artifactId>parent</artifactId>
+		<version>1.3.0</version>
+	</parent>
+	<artifactId>com.shengchen.checkstyle.checker</artifactId>
+	<packaging>jar</packaging>
+	<name>${base.name} :: Checker</name>
+	<dependencies>
+		<dependency>
+			<groupId>com.puppycrawl.tools</groupId>
+			<artifactId>checkstyle</artifactId>
+			<version>8.18</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.shengchen.checkstyle</groupId>
```

Comment:
Why we need this dependency?

Reply:
`checker` need to reference classes in `runner.api`, such as `CheckResult` and `ICheckerService`.

Reply:
ok